roles/smoker/tasks/main.yml

```diff
@@ -4,6 +4,23 @@
   package:
     name: '{{ smoker_browser_packages }}'
 
+- name: 'Install podman'
+  become: true
+  package:
+    name: 'podman'
+
+- name: Set max user namespaces
+  become: true
+  shell: echo 100000 > /proc/sys/user/max_user_namespaces
+
+- name: Set subuid range for user
+  become: true
+  command: usermod --add-subuids 10000-75535 {{ ansible_user }}
```

Comment:
it's super confusing that this is a range, but `/etc/subuid` contains `start:length` instead

Reply:
That's how the command works (with the `-`)

Reply:
Yeah, I realize that, just was commenting for whoever will also look at this and wonder why it does not match their memories ;)